index.js

```diff
@@ -165,9 +165,6 @@ function compile(entrypoint, cache, options) {
     .plugin(tsify, {
       [`${options.typeroots ? '' : '_'}typeRoots`]: [gumTypesDir, localTypesDir],
       target: 'es5',
-      module: 'CommonJS',
-      moduleResolution: 'node',
```

Comment:
At first I thought we should keep these, but that will leave the editor (e.g. VS Code) confused as the `tsconfig.json` won't necessarily have the same values by default. So removing these seems like the right solution.

Reply:
Actually, let's also remove `typeRoots` and the option to skip them. It's also going to create problems. We'll have to bump the major version to release all of this, though, but that's fine.